README.md

```diff
@@ -76,7 +76,29 @@ You can install the extension directly from the source. Follow the following ste
       - Navigate to _about://extensions_.
       - Click on "Load Unpacked" button (make sure you have enabled the _Developer mode_).
       - From the file explorer, choose `ccsearch-browser-extension/dist/opera`.
-
+## Debugging in Development Mode
+   - **Mozilla Firefox**
+      - Navigate to _about:debugging_ and from sidebar select _This Firefox_.
+      - Click on "Load Temporary Add-on" button. ![Load Temporary Add-on](debug-image/debuging-in-firefox.png)
+      - From the file explorer, choose `ccsearch-browser-extension/dist/firefox/manifest.json`.
+      - You will be now able to see CC search browser extension. Now click on the Inspect button. ![Inspect](debug-image/inspect-firefox.png)
+      - A new tab will open with firefox debugger. Now first, click on the _three dots_ and select "Disable Popup Auto-Hide". ![Disable Popup Auto-hide Image](debug-image/three-dots-firefox.png)
+      - Now click on the browser extension from top right corner and the DOM will be loaded with the content to debug. ![Firefox debug](debug-image/click-on-extension.png)
+   - **Google Chrome**
+      - Navigate to _chrome://extensions_.
+      - Make sure that Developer mode is toggled on.
+      - Click on "Load Unpacked" button. ![Load Temporary Add-on](debug-image/chrome-load-unpacked.png)
+      - From the file explorer, choose `ccsearch-browser-extension/dist/chrome`.
+      - You will be now able to see CC search browser extension. Right click on the extension area and select "Inspect". ![Inspect in Chrome](debug-image/inspect-in-chrome.png)
```

Comment:
The screenshot used here is also wrong. The developer must right click on the extension popup to inspect it.

Reply:
@techievivek This issue is still pending.

Reply:
@makkoncept what needs to be changed, can u please explain.
Because I have new screenshot for both Chrome and Opera.

Reply:
The developer needs to right-click on the extension popup and select the "Inspect". You have here pasted the screenshot where you are right-clicking on the `chrome://extension` page. 

![rsz_screenshot_from_2020-02-25_18-03-03](https://user-images.githubusercontent.com/34679965/75248026-48519200-57f9-11ea-8cb7-1cd0aa288e95.png)